### PR TITLE
Fixing app crash when iterating Events without 'kind' prop defined

### LINKED ISSUE #703: App Crash
**Describe the bug**

Workflows tab shows crash:

Error stack:

 
TypeError: Cannot read property 'toLowerCase' of undefined
    at lookupApiLink (webpack-internal:///994:2538:23)
    at renderTableContents (webpack-internal:///994:19244:42)
    at ItemListLayout.getRow (webpack-internal:///994:18545:11)
    at eval (webpack-internal:///994:18793:23)
    at Array.map (<anonymous>)
    at ItemListLayout.renderList (webpack-internal:///994:18792:30)
    at ItemListLayout.render (webpack-internal:///994:18817:58)
    at allowStateChanges (webpack-internal:///9:1008:15)
    at eval (webpack-internal:///11:345:4642)
    at trackDerivedFunction (webpack-internal:///9:802:24)

Component stack:

    in ItemListLayout
    in KubeObjectListLayout
    in Events
    in div
    in WorkloadsOverview
    in Route
    in Switch
    in ErrorBoundary
    in main
    in div
    in MainLayout
    in Workloads
    in Route
    in Switch
    in Switch
    in ErrorBoundary
    in Router
    in Unknown
    in App

**To Reproduce**
Not sure how to reproduce.

**Environment (please complete the following information):**
- Lens Version: 3.5.3
- OS: OSX 10.15.6
- Installation method (e.g. snap or AppImage in Linux): 
 
**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
Your logs go here...
```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
your kubeconfig here
```

**Additional context**

### PR DESCRIPTION
**Pull request by aleksfront**
Related to #703, for some reason Events may come with `involvedObject` prop with undefined `kind` value. It breaks link definition for such an object and, what is more important, totally breaks the app (while reaching `.toLowerCase()` on undefined `kind`). This PR handles the case, allowing to show event without proper link to involved object.

Resolves #703 

Signed-off-by: Alex Andreev <alex.andreev.email@gmail.com>

### CODE REVIEW

File: src/renderer/api/kube-api-parse.ts [hunk 1]
```diff
@@ -121,6 +121,8 @@ export function lookupApiLink(ref: IKubeObjectRef, parentObject: KubeObject): st
     namespace = parentObject.getNs()
   } = ref;
 
+  if (!kind) return "";
```
Comment (ixrock) [inline src/renderer/api/kube-api-parse.ts:124]: Why it might be `falsy`? Where it comes from?

File: src/renderer/components/+events/events.tsx [hunk 2]
```diff
@@ -102,7 +102,7 @@ export class Events extends React.Component<Props> {
             },
             event.getNs(),
             kind,
-            <Link to={detailsUrl} title={name} onClick={stopPropagation}>{name}</Link>,
+            detailsUrl ? <Link to={detailsUrl} title={name} onClick={stopPropagation}>{name}</Link> : name,
```
Comment (ixrock) [inline src/renderer/components/+events/events.tsx]: Maybe there are other places too? Why it's fixed like this?

Comment (ixrock) [reply src/renderer/components/+events/events.tsx]: Maybe the link should be always as "details-link" and handle this exception inside `kube-object-details.tsx`?

File: src/renderer/api/kube-api-parse.ts [hunk 3]
```diff
@@ -112,12 +112,12 @@ export function createKubeApiURL(ref: IKubeApiLinkRef): string {
   }
   return [apiPrefix, apiVersion, namespace, resource, name]
     .filter(v => v)
-    .join("/")
+    .join("/").toLowerCase()
```
Comment (ixrock) [inline src/renderer/api/kube-api-parse.ts]: Why whole path needs to be lower-cased?

**Comment by aleksfront** [comment]
@nevalla Can you review this also? It fixes the issue but exposed that probably function `lookupApiLink()` in `kube-parse-api` should be refactored and double tested.

**Review by nevalla — APPROVED** [review]
LGTM

**Pull request merged**